pkg/satellite/satellitedb/users.go

```diff
@@ -14,15 +14,15 @@ import (
 	"storj.io/storj/pkg/satellite/satellitedb/dbx"
 )
 
-// implementation of User interface repository using spacemonkeygo/dbx orm
+// implementation of Users interface repository using spacemonkeygo/dbx orm
 type users struct {
 	db *dbx.DB
 }
 
 // Get is a method for querying user from the database by id
 func (users *users) Get(ctx context.Context, id uuid.UUID) (*satellite.User, error) {
 
-	userID := dbx.User_Id(id.String())
+	userID := dbx.User_Id([]byte(id.String()))
```

Comment:
In this case you can use `id[:]` rather than `[]byte(id.String())`. For parsing then you can do:

```
func bytesToUUID(data []byte) (uuid.UUID, error) {
	var id uuid.UUID
	n := copy(id[:], data)
	if len(id) != len(data) {
		return uuid.UUID{}, error for invalid bytes
	}
	return id
}
```